packages/xod-project/src/pin.js

```diff
@@ -273,3 +278,53 @@ export const normalizePinLabels = def(
     )
   )
 );
+
+/**
+ * Returns PinLabel by incrementing it in accordance with rules.
+ * - "FOO" -> "FOO2"
+ * - "X7" -> "X8"
+ * - "" -> ""
+ *
+ * Accepting `index` as first argument.
+ * It is `arityLevel` of this Pin, not all Node.
+ * E.G.
+ * Node has `arityLevel === 3` and variadic pin with label 'FOO'.
+ * This function will be called with these arguments:
+ * - 0, 'FOO' -> 'FOO2'
+ * - 1, 'FOO' -> 'FOO3'
+ * - 2, 'FOO' -> 'FOO4'
```

Comment:
A typo? Does it really affect the label for `arityLevel === 0`? By design, it should not.

I mean, I see it indeed affects it. But in the final end-to-end example, I see the very first labels are kept intact. I wonder how it is possible?